packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -676,6 +674,11 @@ const Autocomplete = React.forwardRef(function Autocomplete(inProps, ref) {
                 })}
               </AutocompleteListbox>
             ) : null}
+            {loading && (onScrollToBottom || (!onScrollToBottom && groupedOptions.length === 0)) ? (
+              <AutocompleteLoading className={classes.loading} ownerState={ownerState}>
+                {loadingText}
+              </AutocompleteLoading>
+            ) : null}
```

Comment:
Why was this moved outside of listbox? It is causing the above UI issue mentioned in the demo file.